stratum/hal/lib/tdi/tdi_port_manager.h

```diff
@@ -70,6 +71,26 @@ class TdiPortManager {
 
   // Disables a port.
   virtual ::util::Status DisablePort(int device, int port) = 0;
+
+ protected:
+  // RW mutex lock for protecting the singleton instance initialization and
+  // reading it back from other threads. Unlike other singleton classes, we
+  // use RW lock as we need the pointer to class to be returned.
+  static absl::Mutex init_lock_;
+
+  // Timeout for Write() operations on port status events.
+  static constexpr absl::Duration kWriteTimeout = absl::InfiniteDuration();
+
+  // Default MTU for ports on ES2K.
+  static constexpr int32 kBfDefaultMtu = 10 * 1024;  // 10K
```

Comment:
Hopefully these dont need to be change based on different targets. But we could possibly override in future in derived classes if needed.

Reply:
Good catch. On closer examination, it turns out that the only target that uses this constant is `tofino`.

I've moved the definition from `TdiPortManager` to `TofinoPortManager`.